README.md

```diff
@@ -27,8 +27,42 @@ years of collective experience by the authors of *Clean Code*.
 Inspired from [clean-code-javascript](https://github.com/ryanmcdermott/clean-code-javascript)
 
 ## **Variables**
-### Use meaningful and pronounceable variable names
 
+### Use names that reveal intensions
+
+Everyone who reads your code will thank you.
+
+Example:
+
+```php
+public function getThem(){
+    $list = new ArrayObject([]);
+    foreach($x as $this->theList){
+        if($x[0] == 4){
+            $list->append($x);
+        }
+    }
+    return $list;
+}
+```
```

Comment:
this code is not worked

Reply:
This code fragment is an example about meaningful variables, and although it is somewhat silly, if it were a function of an object, it would work. But if it makes more sense to a 100 functional example, I can do it.

In this example you can know that the function does something, in the second you give an idea that it does something and why. It's part of the clean code book.